README.md

```diff
@@ -126,8 +126,55 @@ Within graphql (the language you query information from and that this plugin put
 
 Keys can be found in Airtable by clicking `Help > API Documentation`.
 
-The API key can be specified in `gatsby-config.js` as noted in the previous section-- **this exposes your key to anyone viewing your repository and is not recommended**.
+The API key can be specified in `gatsby-config.js` as noted in the previous section-- **this exposes your key to anyone viewing your repository and is not recommended. You should inject your API key as recommended below to prevent it from being committed to source control**.
 
-Alternatively, you may specify your API key using an [Environment Variable](https://www.gatsbyjs.org/docs/environment-variables/). This plugin looks for an environment variable called `GATSBY_AIRTABLE_API_KEY` and will use it prior to resorting to the `apiKey` defined in `gatsby-config.js`. You may also specify it in your command line such as `GATSBY_AIRTABLE_API_KEY=XXXXXX gatsby develop`.
+We recommended specifying your API key using an [Environment Variable](https://www.gatsbyjs.org/docs/environment-variables/). This plugin looks for an environment variable called `GATSBY_AIRTABLE_API_KEY` and will use it prior to resorting to the `apiKey` defined in `gatsby-config.js`. You may also specify it in your command line such as `GATSBY_AIRTABLE_API_KEY=XXXXXX gatsby develop`. This method takes advantage of some syntactic sugar courtesy of Gatsby, which automatically makes environment variables prefaced with GATSBY_ available - but any references to environment variables like this that are rendered client-side will **automatically** expose your API key within the browser.
+
+To be safe, you can also setup your API key via a config variable, `apiKey` defined in `gatsby-config.js`. This is the recommended way to inject your API key.
+
+```javascript
+// In gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-source-airtable`,
+    options: {
+      //not prefaced with "GATSBY_", will not automatically be included client-side unless you explicitly expose it
+      apiKey: process.env.AIRTABLE_API_KEY
+      //...etc
+    }
+  }
+]
+```
+
+You can either use a node tool like "dotenv" to load secrets like your Airtable API key from a .env file, or you can specify it in your command line such as `AIRTABLE_API_KEY=XXXXXX gatsby develop`.
 
 If you add or change your API key in an environment variable at the system level, you may need to reload your code editor / IDE for that variable to reload.
+
+### Columns with null values
```

Comment:
```suggestion
### Columns without any values (yet)
```

I feel like "null" carries a bit of extra baggage (depending on your background, or might not even be common knowledge). What do you think about this instead?